afnexus/raid-website
# Docs/updated documentation

## Description
Added documentation to guide the creation of posts as a contributor, as well as added templates for events and posts. 

## Steps to test
1. Check that `template.md` is not rendered as a post in both events and posts page.
2. Check that all blog posts and events posts are rendering normally.

## Linked PRs / Issue
CLOSES #8 
CLOSES #14
